### PR TITLE
New package: lapce 0.2.5

### PR DESCRIPTION
**Pull request by zen0bit**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture x64 glibc and cross build for aarch64

### CODE REVIEW

**Comment by zen0bit** [comment]
Working for me

No problem with updating tree-sitter-julia on my side...

**Comment by jonpikum** [comment]
Apparently the AUR is having the same issue
https://github.com/lapce/lapce/issues/272#issuecomment-1364711123

I also get the same error locally (x86_64)

File: srcpkgs/lapce/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'lapce'
+pkgname=lapce
+version=0.2.5
+revision=1
+archs="aarch64 x86_64"
```
Comment (paper42) [inline srcpkgs/lapce/template:5]: why?

File: srcpkgs/lapce/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+build_style=cargo
+hostmakedepends="cargo cmake pkg-config"
```
Comment (paper42) [inline srcpkgs/lapce/template:7]:
```suggestion
hostmakedepends="cargo pkg-config"
```

File: srcpkgs/lapce/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+makedepends="libgcc expat fontconfig-devel freetype gtk+3-devel glibc libstdc++ libxcb openssl-devel cairo-devel atk-devel gdk-pixbuf-devel"
```
Comment (paper42) [inline srcpkgs/lapce/template:8]:
```suggestion
makedepends="fontconfig-devel gtk+3-devel openssl-devel atk-devel gdk-pixbuf-devel"
```

**Comment by zen0bit** [comment]
Close in favor of https://github.com/void-linux/void-packages/pull/41355

**Pull request closed without merging**